GAE/war/Env.vm

```diff
@@ -12,8 +12,11 @@ loader.register('akvo-flow/flowenv', function(require) {
     mandatoryQuestionID: $env.mandatoryQuestionID,
     showExternalSourcesFeature: $env.showExternalSourcesFeature,
     countries: $env.countries,
-    useGoogleMapsLayers: $env.useGoogleMapsLayers,
-    googleMapsRegionBias: '$env.googleMapsRegionBias'
+    mapsProvider: '$env.mapsProvider',
+    googleMapsRegionBias: '$env.googleMapsRegionBias',
+    cartodbHost: '$env.cartodbHost',
+    hereMapsAppId: '$env.hereMapsAppId',
```

Comment:
If you are adding the properties mentioned previously they will probably need to go here as well, I guess.